setup.py

```diff
@@ -226,13 +226,13 @@ def run(self):
     'pickleshare>=0.4',
     'psutil>=5.3',
     'pygments>=2.0',
-    'pylint>=2.5.0,<3.1',
+    'pylint>=3.1,<4',
     'pylint-venv>=3.0.2',
     'python-lsp-black>=2.0.0,<3.0.0',
     'pyls-spyder>=0.4.0',
     'pyqt5>=5.10,<5.16',
     'pyqtwebengine>=5.10,<5.16',
-    'python-lsp-server[all]>=1.10.0,<1.11.0',
+    'python-lsp-server[all]>=1.11.0,<1.12.0',
```

Comment:
Please update the constraint for development below to be

```python
install_requires.append('python-lsp-server[all]>=1.11.0,<1.13.0')
```

Reply:
Done

Reply:
Now really. Sorry I didn't read your instruction thoroughly.